test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1666,6 +1666,7 @@ def tearDown(self):
 
         for sched in self.scheds:
             self.scheds[sched].cleanup_files()
+        self.server.delete_sched_config()
```

Comment:
is cleanup_files still needed ?

Reply:
@visheshh cleanup_files() clean the temp file in pbs_home directory. So it is  still needed.